src/core/attribute.js

```diff
@@ -98,8 +98,7 @@ export default class Attribute {
   getValue() {
```

Comment:
Is there a usecase where we this method only returns a 'Buffer' ?

Reply:
Yes. But this is an undocumented method in an experimental class. Will keep as is for now and handle it when we make `Attribute` official.